_sass/components/_one-account-banner.scss

```diff
@@ -0,0 +1,28 @@
+.one-account-for-govt {
+  margin: 1rem 0;
```

Comment:
Should the `rem` usage throughout be using USWDS spacing units instead?

For example, from the design, I see a 64px gap, which seems like it should be represented as 8 spacing units.

https://designsystem.digital.gov/utilities/margin-and-padding/

Reply:
Talked to @aduth and we will re-visit this as we look at upgrading the USWDS to the latest version that is leveraged in the login.gov design system.